### PR TITLE
Update descriptions for vpn/hosting provider attributes

### PR DESCRIPTION
**Pull request by lapski**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@
 History
 -------
 
+2.9.1
+++++++++++++++++++
+
+* Updated documentation of anonymizer attributes - is_anonymous_vpn and
```
Comment (oschwald) [inline HISTORY.rst]:
```
``is_anonymous_vpn`` and ``is_hosting_provider``
```

File: geoip2/models.py [hunk 2]
```diff
@@ -328,13 +328,19 @@ class AnonymousIP(SimpleModel):
 
     .. attribute:: is_anonymous_vpn
 
-      This is true if the IP address belongs to an anonymous VPN system.
+      This is true if the IP address is registered to an anonymous VPN
+      provider.
+
+      If a VPN provider does not register subnets under names associated with
+      them, we will likely only flag their IP ranges using the
+      is_hosting_provider attribute.
```
Comment (oschwald) [inline geoip2/models.py]:
```
``is_hosting_provider``
```

File: geoip2/models.py [hunk 3]
```diff
@@ -328,13 +328,19 @@ class AnonymousIP(SimpleModel):
...
 
       :type: bool
 
     .. attribute:: is_hosting_provider
 
-      This is true if the IP address belongs to a hosting provider.
+      This is true if the IP address belongs to a hosting or VPN provider
+      (see description of is_anonymous_vpn attribute).
```
Comment (oschwald) [inline geoip2/models.py]:
```
``is_anonymous_vpn``
```

File: geoip2/models.py (hunk 2 above)
Comment (oschwald) [reply geoip2/models.py]: Similar issues below.

File: HISTORY.rst (hunk 1 above)
Comment (lapski) [reply HISTORY.rst]: Thanks! 44bd306

File: geoip2/models.py (hunk 2 above)
Comment (lapski) [reply geoip2/models.py]: 44bd306

File: geoip2/models.py (hunk 3 above)
Comment (lapski) [reply geoip2/models.py]: 44bd306

**Pull request merged**